lib/libutils/ext/include/util.h

```diff
@@ -62,6 +62,6 @@
 		(type *)((unsigned long)(__ptr) - offsetof(type, member)); \
 	}))
 
-
+#define BIT(nr)			(1UL << (nr))
```

Comment:
This should probably go into a separate commit, but I leave that for Pascal to decide.